### DISCUSSION TITLE
List number of files in a folder?

### DISCUSSION THREAD

**Discussion by danielkrajnik** [eza-community/eza#424] [Q&A, Howto, Help] [answered]
Do you know if there is any way to list number of files contained in each folder? Either recursively or to some finite depth?

![image](https://github.com/eza-community/eza/assets/15096799/0ee77c6f-cc71-41e6-8254-d889122531d4)

**Comment by gierens** [comment, marked as answer]
At the moment neither the size or block size columns display anything for directories. We do however have at least two feature requests for this:

- https://github.com/eza-community/eza/issues/252
- https://github.com/eza-community/eza/issues/389

Probably best to upvote those and continue the discussion there.
